student-info-system/professor/views.py

```diff
@@ -80,9 +81,9 @@ def add_reference(request, sectionid):
                     messages.error(request,
                                    "There was a problem saving the new item to the database.")
                 return render(request, 'professor/reference_add.html', data)
-            section.refresh_reference_items()
+            for section in course.section_set.all():
```

Comment:
are you asking the professor? This will update for all old sections too, not just new/current?

Reply:
changed